src/config.ts

```diff
@@ -87,4 +91,8 @@ export const DeviceRedirects: Record<string, string> = {
     mobile: "exp://192.168.86.24:8081/--/Main",
 };
 
+export const ses = new AWS.SES({
+    region: Config.S3_REGION,
+});
+
```

Comment:
move this into a middleware

Reply:
I had that before but wouldn't that be worse than doing this cuz then there would be a ton of overhead cuz it would have to go through the middleware for every single email that is sent

Reply:
Bc the AWS.SES is called in the library function which only sends one email at a time